src/frequenz/channels/_receiver.py

```diff
@@ -226,7 +226,7 @@ async def receive(self) -> _T_co:
         return received
 
     def map(self, call: Callable[[_T_co], _U_co]) -> Receiver[_U_co]:
-        """Return a receiver with `call` applied on incoming messages.
+        """Map a function on the received values.
```

Comment:
This still has to say "Apply".  The `call` is actually the map function, that does the mapping.  And `map` applies the given map function on the received data.

One possibility:
```suggestion
        """Apply a mapping function on the received values.